docs/2.7.0/docs/query/pqs-user-guide.rst

```diff
@@ -0,0 +1,657 @@
+
+-  All contracts have links (foreign keys) to the transaction in which they were created.
+-  Archived contracts have pointers to the transaction in which they were archived.
+
+Transactions on the ledger are inserted into PostgreSQL concurrently, for high performance. Consistency (for readers) is provided through a watermark mechanism that indicates a consistent offset from which readers can consume for a fully consistent ledger. These details are managed for readers through the functions available in PostgreSQL. Depending on your needs, readers may wish to use or bypass these mechanisms, depending on the type of query and consistency required.
+
+JSON Data
+=========
+
+Relational databases excel at storing structured data for which the schema is known in advance. However, they have traditionally lacked mechanisms for data that is more dynamic or evolves. For example, you may want to store arbitrary Daml contracts and might prefer not to update the database schema every time the underlying template changes.
+
+PostgreSQL helps manage unstructured data through native support for JSON data and allows queries to process this data. For best performance, the PQS stores data as JSONB only.
+
+An example query might look like this:
+
+.. code-block:: none
+
+    SELECT *
+    FROM contract
+    WHERE payload->>'isin' = 'abc123'
+    ORDER BY payload->'issuanceData'->'issueDate'->>'Some';
+
+For more information on querying JSON data, see the section `JSON Functions and Operators <https://www.postgresql.org/docs/12/functions-json.html>`__ in the PostgreSQL manual. The operators ``->``, ``->>``, ``#>``, ``#>>``, and ``@>`` may be of particular interest.
+
+Continuity
+==========
+
+The PQS is intended for continuous operation. Upon restart after an interruption, PQS determines the last consistent offset and continues incremental processing from that point onward. PQS terminates when encountering any error and leaves it up to the orchestration layer (such as Kubernetes) or the operator to determine the appropriate course of action.
+
+High Availability
+=================
+
+Multiple isolated instances of PQS can be instantiated without any cross-dependency. This allows for an active-active high availability, clustering model. Please note that different instances might not be at the same offset due to different processing rates or other factors. After querying one active instance, it is possible for you to see results that are not yet visible on an alternative, active instance. This requires consideration for the client to handle the situation where waiting or a retry is required to service "at least up to" requests.
+
+Setup
+*****
+
+Meeting Prerequisites
+=====================
+
+Here are the prerequisites to run PQS:
+
+-  A PostgreSQL database that can be reached from the PQS. Note that PQS uses the JSONB data type for storing JSON data, which requires Postgres versions 11, 13, and 15 are tested daily.
```

Comment:
@hrischuk-da Need help on this sentence. Not sure what "are tested daily" means?

Reply:
Let's delete the "Are tested daily."